### PR TITLE
Borui Sun hw1

### PR DESCRIPTION
**Pull request by boruis12**
This is my first time using Rmarkdown and pull/push in Github. The TA session and hw01 web page provided a lot of useful resources, including installment, setup and Markdown basics. One major difficulty I encountered is that after uploading a picture to my repo, I struggled a bit to let RStudio know the location of that pic. Everything else is pretty clear and straightforward.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# CFSS-HW01
+                           
+## **Biography of Borui Sun**
```
Comment (chloezhao123) [inline README.md:3]: In the first few lines you have already demonstrated that you know how to use headers and bold in R Markdown!

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+           
+Borui is a second year Master of Public Policy student at the University of Chicago Harris School. She is also serving as a Wilson Fellow at the Energy Policy Institute of UChicago. She obtained her B.A. in political science from the Geroge Washington University. She is interested in pursuing a Ph.D degree in political science after graduate. 
+    
+##### [Borui Sun's Linkedin](https://www.linkedin.com/in/borui-sun-1b2174127/)
+        
+## **Coding Experience**
```
Comment (chloezhao123) [inline README.md:9]: It's impressive to see that you are already familiar with so many programming languages! Your structure here is very clear! I can easily see how proficient you are with each language you mentioned.

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+    + two quarters of studying, familiar with numpy, pandas, BeautifulSoup, Matplotlib packages 
+* **STATA _basic_**:
+    + very basic familarity
+      
+## **Fun Facts**
+* Live with two cats
+* Favoriate color is black
+* Amateur dota player 
+      
+## **Image**
```
Comment (chloezhao123) [inline README.md:25]: I think the entire structure of your first homework assignment resembles that of a resume. After reading it, I have gained information of your academic interest and ability as well as your hobbies. I think the reason I am able to clearly extract so much information from just a few lines is that you have used the list and its indents well. The logic of your markdown file shows through its progression with the help from the use of headers, bold characters, images and URL. I think it is a great first assignment.

File: README.md (hunk 1 above)
Comment (a-aliseda) [inline README.md:3]: Nice use of different headers

File: README.md [hunk 4]
```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# CFSS-HW01
+                           
+## **Biography of Borui Sun**
+           
+Borui is a second year Master of Public Policy student at the University of Chicago Harris School. She is also serving as a Wilson Fellow at the Energy Policy Institute of UChicago. She obtained her B.A. in political science from the Geroge Washington University. She is interested in pursuing a Ph.D degree in political science after graduate. 
+    
+##### [Borui Sun's Linkedin](https://www.linkedin.com/in/borui-sun-1b2174127/)
```
Comment (a-aliseda) [inline README.md:7]: Good use of link to a webpage

File: README.md [hunk 5]
```diff
@@ -1 +1,29 @@
...
+        
+## **Coding Experience**
+While had minimal coding experience beforing coming to Harris, Borui has develoepd a strong interest in data analysis after one year of studying. She had experience with the following programming languages: 
+         
+* **R _proficient_**:
```
Comment (a-aliseda) [inline README.md:12]: Uses different lists and italics and bolds

File: README.md [hunk 6]
```diff
@@ -1 +1,29 @@
...
+* **STATA _basic_**:
+    + very basic familarity
+      
+## **Fun Facts**
+* Live with two cats
+* Favoriate color is black
+* Amateur dota player 
+      
+## **Image**
+![Selfie with one of her cats](https://raw.githubusercontent.com/boruis12/hw01/master/2019-04-02.jpg)
```
Comment (a-aliseda) [inline README.md:26]: The picture loads correctly

**Review by a-aliseda — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   +        |              |            |
| **Coding strategy**         |   +        |              |            |
| **Presentation**            |      +   |              |            |
| **Achievement, creativity** |     +      |              |            |
| **Ease of access**          |     +      |              |            |

+ I really liked your README file because it was straightforward and had all the features they asked
+ I didn't know you could to use bolds and italics in the same word

**Review by chloezhao123 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     +      |              |            |
| **Coding strategy**         |     +      |              |            |
| **Presentation**            |      +    |              |            |
| **Achievement, creativity** |    +       |              |            |
| **Ease of access**          |     +      |              |            |

Remarks:

Some specific praise: 
I think the entire structure of your first homework assignment resembles that of a resume. After reading it, I have gained information of your academic interest and ability as well as your hobbies. I think the reason I am able to clearly extract so much information from just a few lines is that you have used the list and its indents well. The logic of your markdown file shows through its progression with the help from the use of headers, bold characters, images and URL. I think it is a great first assignment.
Something I learned: I learned how to use R markdown to make a neat resume.

File: README.md (hunk 1 above)
Comment (wu-yanru) [inline README.md:3]: Nice use a headers and bold!

File: README.md (hunk 3 above)
Comment (wu-yanru) [reply README.md:25]: Clear structure indicating your proficiency level and detailed understandings regarding various types of coding software, lists and italics and bolds are also been properly conducted.

File: README.md (hunk 1 above)
Comment (wu-yanru) [reply README.md:3]: Clear presentation of the usage of headers:)

**Review by wu-yanru — COMMENTED** [review]
In general, I enjoy reading your biography and get the chance to know you in a more detailed manner. Your biography comprises all the required elements and all of them are been properly embedded in the coding.

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |   x        |              |            |

Remarks:

* One of the best submissions I had to review this time around
* Great use of bullets and levels in the lists
* The image loaded properly and the link to your LinkedIn profile worked
* I've never played DOTA, but I would I would like to try. Does it have a steep learning curve?

**Pull request closed without merging**